terraform/fec_rds.tf

```diff
@@ -105,6 +105,11 @@ resource "aws_db_parameter_group" "fec_default" {
         name  = "log_statement"
         value = "ddl"
     }
+
+    parameter {
+        name  = "max_standby_streaming_delay" # This has no effect on masters, it only affects slaves
+        value = "1 h"
```

Comment:
No concerns on the timeframe, was just verifying!  And as for dynamic parameter, that was in reference to RDS-managed PostgreSQL instances.  Dynamic parameters should not require a reboot to be applied, so this should take effect immediately once it's pushed. :-)